exonum-java-binding-core/src/test/java/com/exonum/binding/blockchain/serialization/TransactionResultSerializerTest.java

```diff
@@ -42,9 +41,10 @@ void roundTrip(TransactionResult expected) {
 
   private static Stream<TransactionResult> testSource() {
     return Stream.of(
-        TransactionResult.valueOf(Type.SUCCESS, null, null),
-        TransactionResult.valueOf(Type.ERROR, 1, "Error description"),
-        TransactionResult.valueOf(Type.UNEXPECTED_ERROR, null, ""));
+        TransactionResult.successful(),
+        TransactionResult.error(1, "Error description"),
+        TransactionResult.error(255, /* No description: */ null),
```

Comment:
`null` description turns into `Optional.empty()` inside the class and then in serializer#toBytes it turns into an empty string, so this test fails :(

Reply:
Thanks, fixed